fast64_internal/oot/data/oot_getters.py

```diff
@@ -0,0 +1,27 @@
+from xml.etree.ElementTree import parse as parseXML, Element
+from .oot_data import OoT_BaseElement
+
+
+def getXMLRoot(xmlPath: str) -> Element:
+    """Parse an XML file and return its root element"""
+    try:
+        return parseXML(xmlPath).getroot()
+    except:
+        from ...utility import PluginError
+
+        raise PluginError(f"ERROR: File '{xmlPath}' is missing or malformed.")
+
+
+def getEnumList(dataList: list[OoT_BaseElement], customItemName: str):
+    """Returns lists containing data for Blender's enum properties"""
+    enumPropItems: list[tuple] = []
+    legacyEnumPropItems: list[tuple] = []  # for older blends
+    customItem = ("Custom", customItemName, "Custom")
+
+    for elem in dataList:
+        enumPropItems.append((elem.key, elem.name, elem.id))
```

Comment:
if we don't use the xml's index to create the items (as "number") then the xml ordering matters for the new object enum items and there's no purpose to the index?

Reply:
when I did this I tried with a messed up XML, like I switched a lot of lines (for instance I had like object number 19 on the first entry) and it worked properly 👍

Reply:
maybe I should remove this function for now, it's a bit useless since the object list need something different